python_bindings/setup.py

```diff
@@ -79,18 +79,19 @@ class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
     c_opts = {
         'msvc': ['/EHsc', '/openmp', '/O2'],
-        'unix': ['-O3'],
+        'unix': ['-O3', '-msse4.2'],
```

Comment:
So - I got a little lazy and skipped that step for Windows. Its currently just building the python extension using the setup.py script. We should definitely add the cmake step for windows - I just find dealing with windows a hassle since I don't have a windows box and can only test by pushing commits to a repo and letting the CI server test it out.

Also I think I broke the windows build with these changes =( It seems like the python unittest is failing in the 'loadIndex' call https://ci.appveyor.com/project/benfred/nmslib